recipes/OpenVisus/meta.yaml

```
@@ -0,0 +1,59 @@
+package:
```

Using a variable for the version will allow the bot to check for updates:
```suggestion
{% set version = "1.3.3" %}

package:
```